recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,2 @@
-python:
-    - 3.6
\ No newline at end of file
+channel_targets:
```

Comment:
Please remove the chanel_targets ad rerender
@adam-urbanczyk

Reply:
Done.